infra/notifications/src/emailSend.js

```diff
@@ -38,6 +38,11 @@ async function messageEmailSend(receivers, sender, messageHash, config) {
     return r.toLowerCase()
   })
 
+  let shortHash = ''
+  if (messageHash && messageHash.startsWith('0x') && messageHash.length > 9) {
+    shortHash = messageHash.slice(2, 7)
```

Comment:
Any reason to use a shortHash rather than full hash to embed in the email ?
It will be hidden content to the user anyway and a few bytes more or less don't matter ?

Reply:
Wasn't initially expecting it to be fully hidden, but you have a point.